bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -118,6 +120,8 @@ pub mod pallet {
 		/// InteriorLocation of this pallet.
 		type PalletLocation: Get<InteriorLocation>;
 
+		type AccountIdConverter: ConvertLocation<Self::AccountId>;
```

Comment:
This shouldn't be necessary if you constrain AccountId with `where <T as frame_system::Config>::AccountId: Into<Location>,`: (e.g. https://github.com/paritytech/polkadot-sdk/pull/8725/commits/0b35bc6b245804ced805d2871092445e7809b289#diff-d0cf69f580d5c5a22cbd36d20464940a91738fbd5c417c779619d82cf5c76abaR293)

Reply:
It's a runtime configuration [here](https://github.com/yrong/polkadot-sdk/blob/b759f5709d5e4138a4a962afd8ca924d8c832127/cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs#L89-L92), so I don't think it will work without being explicitly configured.